### ISSUE TITLE
Frame delta/offset for midi events

### ISSUE THREAD

**Issue by derselbst** [FluidSynth/fluidsynth#126]
Hi, I'm a member of the LMMS development team. I've lately been trying to implement sample-accurate timing for as many of our instruments as possible. This has been implemented by communicating a frameoffset-parameter to instruments when they're asked to play notes.

For our SF2-player which uses Fluidsynth, I'd like to implement the same thing, but so far I haven't found a way to trigger midievents (noteons/noteoffs) with a frame offset in the Fluidsynth API. If there is already a way to do this, then please excuse my ignorance...

So, to recap: now, if I call fluid_synth_noteon, it plays the note always at the start of the period. I'd like to be able to call fluid_synth_noteon with an extra parameter, say: _offset, which would delay the playing of the note by _offset frames, where _offset <= period size. And same for noteoff.

Reported by: diiz

Original Ticket: [fluidsynth/tickets/128](https://sourceforge.net/p/fluidsynth/tickets/128)

**Comment by derselbst** [comment]
I'm understanding this issue as your problem has been solved? Closing for now. Let me know if I was mistaken.
